src/components/BlogForm/BlogForm.jsx

```diff
@@ -30,13 +31,14 @@ const BlogForm = ({ setIsAddBlogFormOpen, blogDetails, onSubmit }) => {
     <Form
       initialValues={blogDetails || null}
       onSubmit={onSubmit}
+      validate={(values) => blogFormValidation(values)}
       render={({ handleSubmit }) => (
         <form onSubmit={handleSubmit}>
-          <div className="title-tags-form-container">
-            <div className="title-tags-form-container__field">
+          <div className="title-tags-form-wrapper">
+            <div>
```

Comment:
```suggestion
            <>
```
we should not use just a `div` if it does not contain anything else with it, as it will add an extra node in the DOM which is redundant, so use `Fragment` instead

Reply:
@maa-wtag bhai, I removed unnecessary `div`